tests/Browser/CreateTaskCategoryTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Tests\Browser;
+
+use App\Models\User;
+use Tests\DuskTestCase;
+use Laravel\Dusk\Browser;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+
+class CreateTaskCategoryTest extends DuskTestCase
+{
+    /**
+     * A Dusk test example.
+     *
+     * @return void
+     */
+    public function testExample()
```

Comment:
Why are all method named `testExample()`? The method name should be based on the thing we are testing in that method.  A better name in this case is `testCreateCategory()`

Reply:
ok sir.